src/Orc.NuGetExplorer.Xaml/ViewModels/ExplorerViewModel.cs

```diff
@@ -131,9 +135,26 @@ private void InitializePages()
             _pageSetup.Values.ForEach(page =>
                 Pages.Add(_typeFactory.CreateInstanceWithParametersAndAutoCompletion<ExplorerPage>(page)));
 
+            foreach (var page in Pages)
+            {
+                page.PropertyChanged += OnPageActiveChanged;
+            }
+
             StartPage = _startPage;
         }
 
+        private void OnPageActiveChanged(object sender, PropertyChangedEventArgs e)
```

Comment:
the name of the method should contain event name 
for instance `OnExplorerPagePropertyChanged()`